index.js

```diff
@@ -24,6 +24,7 @@ const defaults = {
   input: '.combobox',
   list: '.listbox',
   options: '.listbox .option',
+  optgroup: '.listbox .optgroup[data-label]',
```

Comment:
meaning I think here, it should be `optgroup: null` and elsewhere check for it (`if (this.config.optgroup) { // do optgroupy stuff}`)